devops/yarn/pom.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.ray</groupId>
+    <artifactId>ray-on-yarn</artifactId>
+    <version>1.0</version>
+
+    <properties>
+        <hadoop.version>2.8.0</hadoop.version>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-yarn-common</artifactId>
+            <version>${hadoop.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-common</artifactId>
+            <version>${hadoop.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-yarn-client</artifactId>
+            <version>${hadoop.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-yarn-common</artifactId>
+            <type>test-jar</type>
+            <version>${hadoop.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <pluginManagement>
+          <plugins>
+	          <plugin>
+	          <groupId>org.apache.maven.plugins</groupId>
+	          <artifactId>maven-checkstyle-plugin</artifactId>
+	          <version>2.17</version>
+	          <dependencies>
+	              <dependency>
+	                  <groupId>com.puppycrawl.tools</groupId>
+	                  <artifactId>checkstyle</artifactId>
+	                  <version>6.19</version>
+	              </dependency>
+	          </dependencies>
+	          <executions>
+	              <execution>
+	                  <id>validate</id>
+	                  <phase>validate</phase>
+	                  <goals>
+	                      <goal>check</goal>
+	                  </goals>
+	              </execution>
+	          </executions>
+	          <configuration>
```

Comment:
We may need add the style check to CI, you could have a look at test.sh under java package.

Reply:
And whether we could add some unit test case, which prove the code of  deploy on yarn is working, and bind to CI.

Reply:
Ok.